app/views/content/blog/how-to-get-into-teaching.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: How to apply for teacher training
+title: How to get into teaching
```

Comment:
Will need to add a redirect if we are changing the URL

Are there any downsides to having 'how to get into teaching' in the URL/title - will we inadvertently divert search traffic away from the home page to this blog?

Reply:
It's interesting because the home page's title is actually 'inspire the next generation' and only has 'get into teaching' in it because every page has ' | Get into teaching' on the end (which I forgot about so maybe wouldn't be a good idea to have as the title for this blog anyway because then it'd be 'How to get into teaching | Get into teaching').

I'm struggling to think of what to rename the blog to without infringing on another page because the page is so broad and similar to what we've got elsewhere (it's basically a beefed up steps). I've looked on GSC to see what traffic is like and it's only had about 19 clicks in the last six months, a few of which will probably have been me in the last few days. And people are getting to the blog through search terms to do with applications, even though 6 out of the 9 steps aren't even about your application, which is probably not good for users and misleading.

Is the blog even worth keeping? It seems like the sort of thing we have a lot around the site - basically just promoting our support services in a step by step format. Is it a good idea to keep it if we're potentially duplicating info and content, and taking away from other similar, better pages?

Or am I overthinking and we just leave as is/rename as something else?

Reply:
Since 1 Jan, the blog has had just over 1000 unique pageviews, with 246 new users / over 400 sessions as a landing page according to GA, so I think we should probably keep it unless we really have a duplicate content issue?

Plus we don't have a lot of other blog content :-)

Could we use the format used elsewhere
eg Teacher training - 9 steps to get you there

Or something along the lines of
What to do if you are thinking about teaching as a career

Or
From start to finish: How to xxxx

(to try and avoid the keywords used elsewhere)

(those are terrible examples - you could do much better!)

Separately, let's pick up the title/site name issue as part of the routes feedback next week!

Reply:
9 tips to help you apply for teacher training?

I think when it comes to looking more at the blog strategy, maybe one of the first steps should be reviewing what we have and what we might want to get rid of, because I do think we have quite a few very similar blogs that could potentially be competing with each other/producing duplicate content that could be bad for user experience/SEO.

Reply:
Do we need to let the content team know if we're renaming this?

Reply:
Yes - although you can do this once it is in production

You do need to add a redirect to this PR though?